utils/getMultipleCountries.js

```diff
@@ -0,0 +1,35 @@
+const axios = require('axios');
+const numberFormat = require('./numberFormat');
+const exitCountry = require('./exitCountry');
+const to = require('await-to-js').default;
+const handleError = require('cli-handle-error');
+
+module.exports = async (spinner, table, states, countryName, options) => {
+	if (countryName && !states && !options.chart) {
+		const [err, response] = await to(
+			axios.get(`https://corona.lmao.ninja/v2/countries/${countryName}`)
```

Comment:
Isn't this getting just 1 country ? Example:

https://corona.lmao.ninja/v2/countries/Brazil gets data from Brazil only. Shouldn't it stop on "/countries" ?

Click the link to see it: https://corona.lmao.ninja/v2/countries/

Reply:
The issue #54 description states that we need to provide multiple countries to the command line input, for example in this case to test (since not yet deployed) the command : 

`node index.js --s country algeria,france,italy`

Hence we are using the api which allows to get the specified countries input (algeria,france,italy) 

https://corona.lmao.ninja/v2/countries/:countryList

Thanks